.github/CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 /.github/actions/  @elastic/apm-agent-java @elastic/observablt-ci
 /.github/workflows/  @elastic/apm-agent-java @elastic/observablt-ci
 * @elastic/apm-agent-java
```

Comment:
The ordering was wrong when you created this PR, you will need to update it with `main` to get the correct order, the `*` rule must come first.

Also, the `@elastic/ingest-docs` is reported as an error by GH, are you sure the team is publicly visible ?

Reply:
Yep! We use it in the public docs-content repo, for example: https://github.com/orgs/elastic/teams/ingest-docs

Reply:
PTAL